content/en/docs/instrumentation/net/manual.md

```diff
@@ -393,7 +393,9 @@ available by
 
 ## Logs
 
-The logs API & SDK are currently under development.
+The documentation for the logs API & SDK is missing, you can help make it
+available by
+[editing this page](https://github.com/open-telemetry/opentelemetry.io/edit/main/content/en/docs/instrumentation/net/manual.md).
```

Comment:
could temporarily link to https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs as well

Reply:
I didn't know that exists, this could be used to rewrite the docs a lot quicker. Let me take a look and follow up.